README.md

```diff
@@ -109,16 +109,18 @@ The module includes the following response plugins:
   * `options.startHeader` {String} header name for the start time of the request
   * `options.timeoutHeader` {String}  The header name for the time in milliseconds that should ellapse before the request is considered expired.
 
-The module includes the following plugins to be used with restify's after
+The module includes the following plugins to be used with restify's `pre`, `routed`, and `after`
 event, e.g., `server.on('after', plugins.auditLogger());`:
 
 * `auditLogger(options)` - an audit logger for recording all handled requests
   * `options.log` {Object} bunyan logger
   * `[options.server]` {Object} restify server. if passed in, causes server to
      emit 'auditlog' event after audit logs are flushed
-  * `[options.logBuffer]` {Object} optional ringbuffer which is written to if
-     passed in
-  * `[options.printLog]` {Boolean} when true, prints audit logs. default true.
+  * `[options.printLog]` {Boolean} when true, prints audit logs. defaults to true.
```

Comment:
Looks like `printLog` is being used to bypass `log.info`, I agree with this. Going to go ahead and merge since this was the only change requested :smile: